src/components/Menu/index.ts

```diff
@@ -0,0 +1 @@
+export {default, Props, convertGroupToSection} from './Menu';
```

Comment:
Convention for component exports would be

```
export {default as Menu, Props as MenuProps} from './Menu';
```


Is `convertGroupToSection` part of the public API? I can't see it used anywhere. If not we don't need to export it

Reply:
I was using it in a test - but it doesn't actually need to be exported in the `index.ts`. I will remove.